appengine/java_appengine.bzl

```diff
@@ -0,0 +1,289 @@
+# Copyright 2015 The Bazel Authors. All rights reserved.
```

Comment:
is there no way to use 'git mv' to preserve the previous history?

Reply:
This is the main reason I did this as several changes in my branch. It works for the first change which was just a rename.
https://github.com/bazelbuild/rules_appengine/pull/71/commits/30610130c13a1f89916e4b4db53feef29d53e124#diff-3b8b4a89deffca2ab1dabfb178ff7012
but every time I made changes to the moved file, git seems to forget the history. At least during the diff.
https://github.com/bazelbuild/rules_appengine/pull/71/commits/39266aedde387c80ce69bc65886030ee1d304f4d#diff-3b8b4a89deffca2ab1dabfb178ff7012

Running this in my client does show the full history back to 2015.
git log --follow appengine/java_appengine.bzl